.github/workflows/test_pull_requests.yml

```diff
@@ -125,7 +125,7 @@ jobs:
             backend: pyqt6
             coverage: no_cov
           - python: "3.12"
-            platform: macos-latest
+            platform: macos-12
```

Comment:
```suggestion
            platform: macos-13
```

13 should be also x86-64? Or you tested it?

Reply:
https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories

based on this information, the macos-13 means x86 runner and macos-14/macos-latest means arm runner.

Reply:
Checking the GitHub runner-images repo (https://github.com/actions/runner-images?tab=readme-ov-file#available-images) I think `macos-13` means x86 indeed :+1: